.github/workflows/integrationtests.yaml

```
@@ -36,6 +36,7 @@ jobs:
         run: ./gradlew extensions:azure:blobstorage:blob-provision:check
 
   Azure-CosmosDB-Integration-Test:
+
     # run only on upstream repo (requires secret access)
     if: github.repository_owner == 'eclipse-dataspaceconnector'
```

```suggestion
    if: ${{ secrets.COSMOS_KEY }}
```

This way forks can also run this